hyperspy/drawing/utils.py

```diff
@@ -724,10 +749,6 @@ def plot_images(images,
     ax_im_list = [0] * len(isrgb)
     # Loop through each image, adding subplot for each one
     for i, ims in enumerate(images):
-        # Get handles for the signal axes and axes_manager
-        axes_manager = ims.axes_manager
```

Comment:
The purpose of these lines is to create new `axes_manager` for iterating over the signal when there is more than one navigation element. In this case it's done by simply creating an identical signal pointing to the same data.

The purpose of __that__ is to avoid triggering most (maybe even all) of the events when iterating the original `axes_manager`, avoiding:
 - Changing the navigation position (`im.axes_manager.indices`) of the plotted signal. Iterating would leave it in the last index (this is easy)
 - If the original signal is plotted, such iteration would also "flip through" it, thus significantly slowing down everything. (because the original axes_manager has events connected to plot updating) (the main reason)